### PR TITLE
Load all the config values from process.env during build time

### PR DESCRIPTION
**Pull request by msmanoj**
## Type

- [x] Bug fix
- [ ] New feature
- [ ] Improvement to existing feature

## Importance
Required for June-alpha release

## Description
Fixed the webpack config for prod that was considering only the last value specified in the .env file, ignoring the rest.

## Views affected
- Google analytics tracking
- Custom download

### CODE REVIEW

File: src/ensembl/src/services/analytics-service.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import ReactGA from 'react-ga';
 import { AnalyticsType } from 'src/analyticsHelper';
 
-const googleTrackingID = process.env.GOOGLE_ANALYTICS_KEY
-  ? process.env.GOOGLE_ANALYTICS_KEY
+import config from 'config';
+
+const googleTrackingID = config.googleAnalyticsKey
+  ? config.googleAnalyticsKey
```
Comment (azangru) [inline src/ensembl/src/services/analytics-service.ts]: `const googleTrackingID = config.googleAnalyticsKey || '';`

File: src/ensembl/webpack/webpack.config.prod.js [hunk 2]
```diff
@@ -13,9 +13,12 @@ const RobotstxtPlugin = require('robotstxt-webpack-plugin');
 // copy from the environment the same variables that are declared in .env.example
 // NOTE: if no environment variable with corresponding key is present, the value from .env.example will be used
 const dotenv = require('dotenv').config({ path: path.resolve(__dirname, '../.env.example') });
-const getEnvironmentVariables = () => Object.keys(dotenv.parsed).reduce((result, key) => ({
-  [`process.env.${key}`]: JSON.stringify(process.env[key])
-}));
+const environmentVariables = {};
+Object.keys(dotenv.parsed).map((key) => (
+  environmentVariables[`process.env.${key}`] = JSON.stringify(process.env[key])
+));
```
Comment (azangru) [inline src/ensembl/webpack/webpack.config.prod.js]:
```
const getEnvironmentVariables = () => Object.keys(dotenv.parsed).reduce((result, key) => ({
  ...result,
  [`process.env.${key}`]: JSON.stringify(process.env[key])
}));
```

Comment (msmanoj) [reply src/ensembl/webpack/webpack.config.prod.js]: That's done

File: src/ensembl/webpack/webpack.config.prod.js [hunk 3]
```diff
@@ -13,10 +13,10 @@ const RobotstxtPlugin = require('robotstxt-webpack-plugin');
...
 // NOTE: if no environment variable with corresponding key is present, the value from .env.example will be used
 const dotenv = require('dotenv').config({ path: path.resolve(__dirname, '../.env.example') });
-const environmentVariables = {};
-Object.keys(dotenv.parsed).map((key) => (
-  environmentVariables[`process.env.${key}`] = JSON.stringify(process.env[key])
-));
+const getEnvironmentVariables = () => Object.keys(dotenv.parsed).reduce((result, key) => ({
+  ...result,
+  [`process.env.${key}`]: JSON.stringify(process.env[key])
+}));
```
Comment (azangru) [inline src/ensembl/webpack/webpack.config.prod.js]: Start with empty object (on line 19).

File: src/ensembl/src/services/analytics-service.ts (hunk 1 above)
Comment (msmanoj) [reply src/ensembl/src/services/analytics-service.ts]: That's done

File: src/ensembl/webpack/webpack.config.prod.js (hunk 3 above)
Comment (msmanoj) [reply src/ensembl/webpack/webpack.config.prod.js]: Done

**Pull request merged**